src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  height: 100vh;
+  padding: $padding;
+  margin: 0 auto;
+  width: ($day-box-size + $gap-between-boxes) * $columns + ($padding * 2);
```

Comment:
create variable for this value

btw, what is 2?

Reply:
There are two puddings on the left and on the right) Its better to make another variable for this?